cirq-core/cirq/contrib/paulistring/optimize.py

```diff
@@ -12,25 +12,36 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import Sequence
+from typing import Callable
 
-from cirq import ops, circuits, optimizers
+from cirq import ops, circuits, transformers
 from cirq.contrib.paulistring.pauli_string_optimize import pauli_string_optimized_circuit
 from cirq.contrib.paulistring.clifford_optimize import clifford_optimized_circuit
 
 
+class CZTargetGatesetWithPostCleanup(transformers.CZTargetGateset):
```

Comment:
Can we make this `_CZTargetGateSet` and indicate in the docstring it is private ?

Reply:
done